README.md

```diff
@@ -1,289 +1,608 @@
-  services:
-    - group: postgres.dev
-      kind: Service
-      name: global-user-db
-      version: v1beta1
-      id: postgres-global-user
-    - group: ibmcloud.ibm.com
-      version: v1alpha1
-      kind: Binding
-      name: coligo-service-binding
-      id: coligo-service-binding
-  application:
-    name: nodejs-rest-http-crud
-    group: apps
-    version: v1
-    resource: deployments
+  name:                 # string, optional, default: .metadata.name
+  kind:                 # string, optional
+  provider:             # string, optional
+
+  application:          # PodSpec-able resource ObjectReference
+    apiVersion:         # string
+    kind:               # string
+    name:               # string
+    containers:         # []intstr.IntOrString, optional
+    ...
+
+  service:              # Provisioned Service-able resource ObjectReference
+    apiVersion:         # string
+    kind:               # string
+    name:               # string
+    ...
+
+status:
+  conditions:           # []Condition containing at least one entry for `Ready`
+  - type:               # string
+    status:             # string
+    lastTransitionTime: # Time
+    reason:             # string
+    message:            # string
 ```
 
-#### Customizing data bindings
+## Example Resource
 
-The `ServiceBinding` CR has an element per service called `dataMappings`, whereby the author of the CR can rename certain binding items and/or compose of multiple items:
+```yaml
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: online-banking-to-account-service
+spec:
+  name: account-service
 
-Partial sample of service-specific mappings:
-```
-  services:
-    - group: postgres.dev
-      kind: Service
-      name: global-user-db
-      version: v1beta1
-      id: postgres-global-user
-    - group: ibmcloud.ibm.com
-      version: v1alpha1
-      kind: Binding
-      name: watson-service-binding
-      id: watson-service-binding
-      dataMappings:
-        - name: WATSON_URL
-          value: {{  .status.host }} / {{ .status.port }}
-        - name: WATSON_USERNAME
-          value: {{  .status.username }}
+  application:
+    apiVersion: apps/v1
+    kind:       Deployment
+    name:       online-banking
+
+  service:
+    apiVersion: com.example/v1alpha1
+    kind:       AccountService
+    name:       prod-account-service
+
+status:
+  conditions:
+  - type:   Ready
+    status: True
 ```
 
-#### <kbd>EXPERIMENTAL</kbd> Synthetic data bindings
+## Reconciler Implementation
 
-If a `dataMappings` requires a cross-service composition then a new synthetic service entry must be created.  
+A Reconciler implementation for the `ServiceBinding` type is responsible for binding the Provisioned Service binding `Secret` into an Application.  The `Secret` referred to by `.status.binding.name` on the resource represented by `service` **MUST** be mounted as a volume on the resource represented by `application`.
```

Comment:
```suggestion
A Reconciler implementation for the `ServiceBinding` type is responsible for binding the Provisioned Service binding `Secret` into an Application.  The `Secret` referred to by `.status.binding.name` on the resource represented by `service` **MUST** be mounted as a volume on the resource represented by `application`.   If the `application` is owned by a custom controller it is up to the implementation to decide how to ensure this data remains persisted, such as utilizing a mutation webhook or directly acting upon the `Secret`.  
```

Reply:
How would directly acting upon the Secret make a difference in this case? The volume mount is going to be removed from the controlled resource.

Perhaps we should remove everything after the "such as" because it leads to more questions than answers.

Reply:
This happens today where the "implementation" is a combination of the Runtime Component Operator (RCO) and the Service Binding Operator (SBO).  The `ServiceBinding` CR is processed / watched by SBO, who creates the Secret.  But the `application` is deployed and owned by RCO.  So RCO directly mounts the Secret that SBO creates after it processes the request, since it (RCO) owns the deployment.

Reply:
I'm up for this change, removing the example and leaving it up exclusively to implementations to choose.

Reply:
Addressed via 511e8bf